packages/web3-constants/evm/explorer.json

```diff
@@ -37,6 +39,8 @@
         "Celo": "https://explorer.celo.org/api",
         "Fantom": "https://api.ftmscan.com/api",
         "Aurora": "https://explorer.mainnet.aurora.dev/api",
-        "Aurora_Testnet": "https://explorer.testnet.aurora.dev/api"
+        "Aurora_Testnet": "https://explorer.testnet.aurora.dev/api",
+        "Harmony": "",
```

Comment:
https://explorer.harmony.one/
You should self review after you submitting a PR.